src/framework/theme/components/window/window.component.ts

```diff
@@ -32,16 +32,16 @@ import { NbWindowRef } from './window-ref';
         <div cdkFocusInitial class="title" tabindex="-1">{{ config.title }}</div>
 
         <div class="buttons">
-          <button class="button" (click)="minimize()">
+          <button nbButton appearance="ghost" (click)="minimize()">
```

Comment:
Just `ghost` like here https://github.com/akveo/nebular/pull/1388/files#diff-4fc82888340ae1f2fcb079d34473990bR16?
Or vice-versa? Either way, it seems that one of these is misused.

Reply:
Same like in calendar. Forgot that I can set it via attribute.
Fixed in 6cfa17f.